neqo-transport/src/connection/mod.rs

```diff
@@ -2817,6 +2811,36 @@ impl Connection {
         }
     }
 
+    fn pad_initial(
+        &self,
+        encoder: &mut Encoder<&mut Vec<u8>>,
+        initial: &mut sent::Packet,
+        profile: &SendProfile,
+    ) {
+        if encoder.len() >= profile.limit() {
+            return;
+        }
+
+        qdebug!(
+            "[{self}] pad Initial from {} to {}",
+            encoder.len(),
+            profile.limit()
+        );
+        let pad_amount = profile.limit() - encoder.len();
+        initial.track_padding(pad_amount);
+        // This ensures that the last bytes are a SCONE indication, if there is enough space.
+        // This is not tracked, other than for congestion control (above)
+        if pad_amount >= Self::SCONE_INDICATION.len() {
+            encoder.pad_to(
+                profile.limit() - Self::SCONE_INDICATION.len() + 1,
+                Self::SCONE_INDICATION[0],
+            );
+            encoder.encode(&Self::SCONE_INDICATION[1..]);
```

Comment:
Do you expect `SCONE_INDICATOR` to increase in length in the future?

Reply:
I'm taking precautions for now.  I expect that this will be reduced to a single assignment by the compiler.